pattern-2/README.md

```diff
@@ -0,0 +1,120 @@
+# Pivotal Cloud Foundry Resources for WSO2 Enterprise Integrator
+
+This repository contains resources required to build and install WSO2 Enterprise Integrator in a Pivotal Cloud Foundry (PCF) environment. This document provides instructions you need to follow in order to deploy a WSO2 Enterprise Integrator setup on PCF.
+
+## Prerequisites
+Before starting the installation process, ensure that the following prerequisites are completed:
+- A pre-configured Pivotal environment: For more information on the process for setting up the Pivotal environment on AWS, Azure, and GCP, see the documentation provided by Pivotal in [Architecture and Installation Overview](https://docs.pivotal.io/pivotalcf/2-4/installing/index.html).
+- An SQL Database (MySQL or MS SQL): This database should contain the tables required to run Enterprise Integrator. The database schema required to populate the tables can be found within the <APIM_HOME>/dbscripts/ directory, where <APIM_HOME> refers to the [Enterprise Integrator Binary](https://wso2.com/api-management/).
+- BOSH [CLI](https://bosh.io/docs/cli-v2/).
+- PCF Tile Generator [CLI](https://docs.pivotal.io/tiledev/2-3/tile-generator.html).
+
+## Setting up for BOSH Release
+The first step in running Enterprise Integrator on PCF is creating a BOSH release. The following set of instructions should be followed in this process:
+1. Clone the PCF Enterprise Integrator repository on GitHub by issuing the following command.
+    ```bash
+    git clone https://github.com/wso2/pivotal-cf-ei.git
+    ```
+    Enterprise Integrator contains five main components named Publisher, Store, Gateway, Traffic Manager, and Key Manager. In a stand-alone APIM setup, these components are deployed in a single server. However, in a typical production setup, they need to be deployed in separate servers for better performance. Installing and configuring each or selected component/s in different servers is known as a distributed setup.
```

Comment:
APIM does not have components named Publisher, Store, etc. Remove and add a proper description.